datadog_checks_dev/datadog_checks/dev/tooling/templates/integration/jmx/{check_name}/datadog_checks/{check_name}/data/conf.yaml.example

```diff
@@ -121,17 +121,17 @@ instances:
     #
     # rmi_registry_ssl: false
 
-    ## @param rmi_connection_timeout - number - optional - default: 30
-    ## The connection timeout, in seconds, when connecting to a remote JVM.
+    ## @param rmi_connection_timeout - number - optional - default: 20000
+    ## The connection timeout, in milliseconds, when connecting to a remote JVM.
```

Comment:
Shall we just drop conf.yaml.example and generate it at integration creation from config spec ?